packages/cms/pages/auth/register/register.tsx

```diff
@@ -31,36 +49,58 @@ export const Register: React.FunctionComponent = () => {
 
       <EuiSpacer size="xl" />
 
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiFormRow label="First name">
-            <EuiFieldText autoFocus />
-          </EuiFormRow>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiFormRow label="Last name">
-            <EuiFieldText />
-          </EuiFormRow>
-        </EuiFlexItem>
-      </EuiFlexGroup>
+      <form method="post" onSubmit={submitRegistration}>
```

Comment:
A better practice for naming even listeners would be `onSubmit` instead of `submitRegistration`

Reply:
Thanks